src/app/public/modules/action-hub/action-hub.component.ts

```diff
@@ -0,0 +1,34 @@
+import { Component, Input } from '@angular/core';
+
+import { SkyLink } from '../link-list/types/link';
+import { SkyActionHubNeedsAttention } from '../needs-attention/types/action-hub-needs-attention';
+
+import { SkyActionHubData } from './types/action-hub-data';
+
+let data: SkyActionHubData;
+
+@Component({
```

Comment:
We need JSDocs comments here for documentation. (Both on the component itself and on the `Input`)

Example: https://github.com/blackbaud/skyux-lists/blob/master/src/app/public/modules/repeater/repeater-item.component.ts